### ISSUE TITLE
can you provide the training code?

### ISSUE THREAD

**Issue by seyeeet** [wvangansbeke/Unsupervised-Semantic-Segmentation#9]
thanks for your work.
can it be possible to share the training code?

**Comment by wvangansbeke** [comment]
Hi @seyeeet 

The instructions to train the model can be found in the README. The training code itself is located in the `pretrain/` dir. Finetuning/clustering for the segmentation task can be found in the `segmentation/` dir.
I hope this is clear.

**Comment by seyeeet** [comment]
Hi @wvangansbeke 
Thanks for clarification, so I have everything to train and evaluate the model on a custom dataset?

**Comment by wvangansbeke** [comment]
Hi @seyeeet 

You will need to write your own dataloader and make it compatible with the rest of the code. You might also need to change some hyperparameters, e.g., learning rate.
